### ISSUE TITLE
Move Everything.agda to src.

### ISSUE THREAD

**Issue by anuyts** [agda/agda-categories#342]
Otherwise it does not seem to typecheck. I get the following error message:
```
The name of the top level module does not match the file name. The
module Everything should be defined in one of the following files:
```
but I think it means that Agda has detected that the file Everything.agda is almost but not quite part of a library.
It does typecheck if I move it into `src` but also if I move it to a completely different location.

**Comment by HuStmpHrrr** [comment]
Everything is not supposed to be in `src`. stdlib's README also won't load. but indeed there should be a better solution.

**Comment by anuyts** [comment]
What is the reason for the statement "Everything is not supposed to be in `src`"?

**Comment by HuStmpHrrr** [comment]
because it is not useful as it just imports all files in the library. it's there just for typechecking the whole library by ci.

**Comment by jamesmckinna** [comment]
Indeed, stdlib does not check in `Everything.agda`, it is auto-generated by `GenerateEverything.hs` (which caught me out when I recently did the bulk renaming from `Category.*` to `Effect.*` (issue [agda/agda-stdlib#1636](https://github.com/agda/agda-stdlib/issues/1636)). The status of `README.agda` is indeed also anomalous, but perhaps worth considering some sort of auto-generation for your `Everything.agda`... even though such an indirection then potentially disguises that dependency when global changes are made.

**Comment by JacquesCarette** [comment]
So it seems that we shouldn't check it in. I'll do that.

**Comment by JacquesCarette** [comment]
Closed by [8643687](https://github.com/agda/agda-categories/commit/8643687021b321420665bdf7f9a27983b564291b)

**Comment by anuyts** [comment]
Then again, type-checking Everything.agda is an easy way to check if the library works with your current Agda installation.
